### PR TITLE
Add metrics for configcenter

### PR DESCRIPTION
**Pull request by fomeiherz**
## What is the purpose of the change
Link issue: https://github.com/apache/dubbo/issues/11147

## Brief changelog
Add metrics for configcenter.

### CODE REVIEW

**Comment by AlbumenJ** [comment]
image

File: dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/ConfigCenterMetricsCollector.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+
+import static org.apache.dubbo.metrics.model.MetricsCategory.CONFIGCENTER;
+
+/**
+ * @author guiyi.yuan
+ * @date 2/8/23 10:06 PM
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/ConfigCenterMetricsCollector.java:37]: Remove these tags pls

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Comment by songxiaosheng** [comment]
please fix ci problem 
```
Error:  /home/runner/work/dubbo/dubbo/dubbo/dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/ConfigCenterMetricsCollector.java:23:8: Unused import - org.apache.dubbo.metrics.model.ApplicationMetric. [UnusedImports]
Error:  /home/runner/work/dubbo/dubbo/dubbo/dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/ConfigCenterMetricsCollector.java:25:8: Unused import - org.apache.dubbo.metrics.model.MetricsCategory. [UnusedImports]
Audit done.
[INFO] There are 2 errors reported by Checkstyle 8.41 with codestyle/checkstyle.xml ruleset.
Error:  src/main/java/org/apache/dubbo/metrics/collector/ConfigCenterMetricsCollector.java:[23,8] (imports) UnusedImports: Unused import - org.apache.dubbo.metrics.model.ApplicationMetric.
Error:  src/main/java/org/apache/dubbo/metrics/collector/ConfigCenterMetricsCollector.java:[25,8] (imports) UnusedImports: Unused import - org.apache.dubbo.metrics.model.MetricsCategory.
```

**Comment by songxiaosheng** [comment]
please fix dependency cycle
image

**Comment by AlbumenJ** [comment]
image
Still failed

**Comment by songxiaosheng** [comment]
> image
> Still failed

we need Extract qos dependencies from the metrics module and extend the implementation to common dependencies
 to  avoid dependency cycles

**Review by songxiaosheng — APPROVED** [review]
LGTM

**Comment by songxiaosheng** [comment]
@AlbumenJ  PATL

**Review by AlbumenJ — APPROVED** [review]
LGTM

**Pull request merged**